src/Export_Command.php

```diff
@@ -315,6 +324,18 @@ private function check_post__in( $post__in ) {
 		return true;
 	}
 
+	private function check_with_attachments( $with ) {
+		if ( is_null( $with ) )
+			return true;
+
+		if ( (int) $with <> 0 && (int) $with <> 1 ) {
```

Comment:
yeah but currently, all option use a "_" between words, so that would mean --no-with_attachments

Reply:
Hmm, good point. I think the underscore in this command came about because it is using some of the variable names from WP.

I'll discuss this in tomorrow's office hours to see what the best approach is.

Reply:
I discussed this with @danielbachhuber , and we agreed to use `--with_attachments` as flag, knowing that the negation will be a weird `--no-with_attachments`.

Reply:
Do you need assistance with converting the parameter into flag notation?